src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -134,7 +135,7 @@ final case class EServicesApiServiceImpl(
         isSubscribed = agreement.exists(a => SUBSCRIBED_AGREEMENT_STATES.contains(a.state)),
         activeDescriptor =
           getActiveDescriptor(eService).map(ad => CompactDescriptor(ad.id, ad.state.toApi, ad.version)),
-        mail = requesterTenant.mails.headOption.map(_.toApi)
+        mail = producerTenant.mails.headOption.map(_.toApi)
```

Comment:
I left the current logic and did not filter by `kind = 'CONTACT_EMAIL'`, but let me know

Reply:
atm the only kind of mail that is possible to have is this, it's even placed in a empty list in some deser throughout our codebase

Reply:
On a second thought, if it is the exact kind of email we want to return, I would add the filter to avoid that the addition of a new email kind will return the wrong information tomorrow